deeplearning/ml4pl/models/ggnn/ggnn.py

```diff
@@ -210,12 +221,18 @@ def __init__(self, *args, **kwargs):
       pretrained_embeddings=inst2vec_embeddings,
       test_only=FLAGS.test_only,
     )
+    app.Log(
+      1,
+      "Using device %s with dtype %s",
+      self.model.dev,
+      torch.get_default_dtype(),
+    )
 
     if DEBUG:
       for submodule in self.model.modules():
         submodule.register_forward_hook(nan_hook)
 
-    self.model.to(self.dev)
+    self.model.to(self.model.dev)
```

Comment:
this line shouldn't exist. it's old code and superceded by the model moving itself to model.dev before instanciating an optimizer!

Reply:
Should the entire line be removed, or just rever the diff to `self.model.to(self.dev)`?